tests/test_probe.py

```diff
@@ -13,6 +18,33 @@ def test_has_binary():
 def test_has_no_binary():
     assert not probe.has_binary('foo')
 
-
 def test_has_test_runner():
     assert probe.has_test_runner()
+
+def test_accepts_readme():
+	os.chdir(context.module_name)
```

Comment:
Could you change this to use the `plumbum.local.cwd` context manager?
Like https://github.com/michaeljoseph/changes/blob/master/tests/__init__.py#L35